project/backend/src/test/java/com/google/sps/TestRestaurantParser.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps;
+
+import com.google.sps.data.JsonToRestaurantParser;
+import com.google.sps.data.Restaurant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class TestRestaurantParser {
```

Comment:
file naming nit: it should be named ClassNameTest 
so in this case JsonToRestaurantParserTest.java